TWLight/users/models.py

```diff
@@ -256,6 +223,212 @@ def encode_wp_username(self, username):
         result = urllib.parse.quote(username)
         return result
 
+    @property
+    def wp_editcount(self):
+        """
+        Fetches latest editcount from EditorLogs related to this editor.
+        Parameters
+        ----------
+        self
+
+        Returns
+        -------
+        int : Most recently recorded Wikipedia editcount
+        """
+        try:
+            return (EditorLog.objects.filter(editor=self).latest("timestamp")).editcount
+        except models.ObjectDoesNotExist:
+            pass
+
+    @property
+    def wp_editcount_updated(self):
+        """
+        Fetches timestamp of latest editcount from EditorLogs related to this editor.
+        Parameters
+        ----------
+        self
```

Comment:
Maybe we should remove this, I think docstrings never include `self` params :)

Reply:
That's weird, as I just used the auto-docstring function in pycharm. Those were there in some places, but not others. I had a bunch of editor updates that I applied midway through, and it doesn't seem to be doing it now. Definitely doesn't make any sense. Gone!